csu

```diff
@@ -102,6 +105,30 @@ function shell {
   docker-compose run django bash
 }
 
+# Run style checks
+function style {
+  echo "Running PEP8 style checker..."
+  docker-compose run django /docker_venv/bin/flake8
+  echo "Running Python docstring checker..."
+  docker-compose run django /docker_venv/bin/pydocstyle --count --explain
+}
+
+# Run test suite with coverage checking
+function test_suite {
+  echo "Running test suite..."
+  docker-compose run django /docker_venv/bin/coverage run --rcfile=.coveragerc ./manage.py test --pattern "test_*.py" -v 3
+  docker-compose run django /docker_venv/bin/coverage report
```

Comment:
This just outputs the coverage graph using `coverage`, but I've yet to figure out how you uploaded that coverage data using `coveralls`, is there supposed to be a `.coveralls.yml` file?